generators/2.0/java/index.js

```diff
@@ -0,0 +1,160 @@
+const Generator = require('yeoman-generator');
+const colors = require('colors');
+const path = require('path');
+const keyPair = require('keypair');
+
+const BASE_JAVA = 'src/main/java';
+const BASE_RESOURCES = 'src/main/resources';
+
+module.exports = class extends Generator {
+
+    initializing () {
+
+    }
+
+    async prompting () {
+        this.answers = await this.prompt([
+            {
+                type: 'list',
+                name: 'build',
+                message: 'Select your build system',
+                choices: [
+                    'Maven',
+                    'Gradle',
+                ]
+            },
+            // {
```

Comment:
Comment or not comment ?

Reply:
Comment :)

Reply:
If I have time I'd like to add the ability to select the SpringBoot integration as well. But I need to update our SpringBoot starter before